src/content/docs/style-guide/writing-strategies/introduction-style-guide.mdx

```diff
@@ -36,3 +36,10 @@ Find the information you need:
 * For our writing strategy, see docs in this section on [voice](/docs/style-guide/writing-strategies/voice-strategies-docs-sound-new-relic/), [five questions for great content](/docs/style-guide/writing-strategies/five-questions-help-write-docs/), [how to organize your content](/docs/style-guide/writing-strategies/organize-doc), and more.
 * For a quick reference for terms usage, see our [Usage dictionary](/docs/style-guide/word-choice/usage-dictionary).
 * For how to write and edit pages on docs.newrelic.com, see [Create and edit content](/docs/style-guide/writing-docs/processes-procedures/create-edit-content).
+
+If you're new to editing New Relic's Docs, here's a short video that will walk you through the process step by step:
```

Comment:
@homelessbirds since this video is about that last bullet point, how about just indenting it right under that point? You could maybe extend the bullet point like this:

* For how to write and edit pages on docs.newrelic.com, see [Create and edit content](/docs/style-guide/writing-docs/processes-procedures/create-edit-content). or watch this video that is less than two minutes:

Reply:
That's a great idea, thanks!